package.json

```diff
@@ -17,7 +17,11 @@
     "name": "hashtree",
     "triples": {
       "additional": [
-        "aarch64-unknown-linux-gnu"
+        "x86_64-apple-darwin",
+        "aarch64-apple-darwin",
+        "x86_64-unknown-linux-gnu",
```

Comment:
can you also include "aarch64-unknown-linux-gnu" (arm linux support)

Reply:
OK.  linux arm64 works now.  Just need to get windows working properly and we are good to go